bin/beeline

```diff
@@ -25,7 +25,7 @@ set -o posix
 
 # Figure out if SPARK_HOME is set
 if [ -z "${SPARK_HOME}" ]; then
-  export SPARK_HOME="$(cd "`dirname "$0"`"/..; pwd)"
+  source `dirname $0`/find-spark-home
```

Comment:
```
In bin/beeline line 28:
  source `dirname $0`/find-spark-home
  ^-- SC1090: Can't follow non-constant source. Use a directive to specify location.
         ^-- SC2046: Quote this to prevent word splitting.
         ^-- SC2006: Use $(..) instead of legacy `..`.
                  ^-- SC2086: Double quote to prevent globbing and word splitting.
```